r/R/dplyr.R

```diff
@@ -1099,3 +1126,12 @@ not_implemented_for_dataset <- function(method) {
     call. = FALSE
   )
 }
+
+#' Does this string contain regex metacharacters?
+#' 
+#' @param string String to be tested
+#' @keywords internal
+#' @return Logical: does `string` contain regex metacharacters?
+contains_regex <- function(string){
+  grepl("[.\\|()[{^$*+?]", string)
+}
```

Comment:
Nice! Please add a newline at the end of the file:
```suggestion
}

```